soc/nxp/imx/imx9/Kconfig.soc

```diff
@@ -1,32 +1,11 @@
-# Copyright 2022,2024 NXP
```

Comment:
Don't remove 2022?

Reply:
I moved files under imx9 to imx9/imx93. So, this is a actually a new file I write manually.

Reply:
okay, thanks.